client-rest/build.gradle

```diff
@@ -94,4 +96,39 @@ task installTools {
     }
 }
 
+// creates the distribution ZIP containing the compiled Angular application
+task createDistribution(type: Zip) {
+    from "dist/portability-demo"
+    include "*"
+    include "*/*"
+    archiveName "client-rest.zip"
+    destinationDir(file("${buildDir}/libs"))
+}
+
+def distFile = file("${buildDir}/libs/client-rest.zip")
```

Comment:
Does this only publish the distribution directory?

Reply:
The build creates an Angular-compiled web app, zips it and publishes a ZIP-based artifact to Maven/JCenter. That will allow any Gradle, Maven or Ivy-based build system to use built-in dependency resolution to download it as a standard artifact. The starter repo will do this and then unzip the contents to the Docker image.

Reply:
Does this include sources or only the compiled code? I'm not sure how javascript distributions work.

Reply:
Oh and can we add a comment or documentation